prac_05/state_names.py

```diff
@@ -0,0 +1,27 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
```

Comment:
I believe that the dictionary contents need to be printed formatted as per the prac instructions:

"""Write a loop that prints all the states and names neatly lined up with string formatting, like:

`NSW is New South Wales
QLD is Queensland
NT  is Northern Territory`

Reply:
Thank you, I can't believe I missed that one

Reply:
Bahaha no worries, it can be easy to miss stuff, especially if you are swapping between tabs.